data/maintainersData.ts

```diff
@@ -1,7 +1,164 @@
 export const maintainersData = [
   { login: 'rupali-codes', name: 'Rupali Haldiya' },
   { login: 'CBID2', name: 'Christine Belzie' },
-  { login: 'ujjawaltyagii', name: 'Ujjawal Tyagi' },
-  { login: 'k-deepak04', name: 'Deepak Singh' },
   { login: 'Anmol-Baranwal', name: 'Anmol Baranwal' },
+  { login: 'aftabrehan', name: 'Aftab Rehan' },
+]
+
+export interface MaintainersDataInterface {
+  name: string
+  avatarUrl: string
+  login: string
+  designation: string
+  role: string
+  firstLink: string
+  firstTxt: string
+  secondLink: string
+  secondTxt: string
+  thirdLink: string
+  thirdTxt: string
+}
+
+export const Maintainers: MaintainersDataInterface[] = [
+  {
+    name: 'Rupali Haldiya',
+    avatarUrl: 'https://avatars.githubusercontent.com/u/78981177?v=4',
+    login: 'rupali-codes',
+    designation: 'FullStack Developer',
+    role: 'Founder',
+    firstLink: 'https://github.com/sponsors/rupali-codes',
+    firstTxt: 'Sponsor',
+    secondLink: 'https://github.com/rupali-codes',
+    secondTxt: 'GitHub',
+    thirdLink: 'https://twitter.com/rupali_codes',
+    thirdTxt: 'Twitter',
+  },
+  {
+    name: 'Anmol Baranwal',
+    avatarUrl: 'https://avatars.githubusercontent.com/u/74038190?v=4',
+    login: 'Anmol-Baranwal',
+    designation: 'Software Developer',
+    role: 'Maintainer',
+    firstLink: 'https://www.linkedin.com/in/Anmol-Baranwal',
+    firstTxt: 'LinkedIn',
+    secondLink: 'https://github.com/Anmol-Baranwal',
+    secondTxt: 'GitHub',
+    thirdLink: 'https://twitter.com/Anmol_Codes',
+    thirdTxt: 'Twitter',
+  },
+  {
+    name: 'Christine Belzie',
+    avatarUrl: 'https://avatars.githubusercontent.com/u/105683440?v=4',
+    login: 'CBID2',
+    designation: 'Technical Writer',
+    role: 'Maintainer',
+    firstLink: 'https://github.com/sponsors/CBID2',
+    firstTxt: 'Sponsor',
+    secondLink: 'https://www.linkedin.com/in/christinebelzie',
+    secondTxt: 'LinkedIn',
+    thirdLink: 'https://twitter.com/CodesChrissy',
+    thirdTxt: 'Twitter',
+  },
+  {
+    name: 'Aftab Rehan',
+    avatarUrl: 'https://avatars.githubusercontent.com/u/93012310?v=4',
+    login: 'aftabrehan',
+    designation: 'UI/UX Designer',
+    role: 'Maintainer',
+    firstLink: 'https://aftabrehan.com',
+    firstTxt: 'Website',
+    secondLink: 'https://github.com/aftabrehan',
+    secondTxt: 'GitHub',
+    thirdLink: 'https://twitter.com/aftabrehan_',
+    thirdTxt: 'Twitter',
+  },
+]
+
+export interface Contributor {
+  twitter_username?: string
+  id: number
+  avatar_url: string
+  name: string
+  login: string
+  contributions: number
+}
+
+export const dummyContributors: Contributor[] = [
```

Comment:
Nope, I left it because more often than not, the GitHub API limit is reached, so anyone can use `dummyContributors`.

It's just for people to test it, regardless of the API limit.